net-mail/qmail-notify/qmail-notify-0.93-r2.ebuild

```
@@ -12,9 +12,7 @@ HOMEPAGE="http://untroubled.org/qmail-notify/"
 SLOT="0"
 LICENSE="GPL-2"
 KEYWORDS="amd64 ~hppa sparc x86"
-IUSE=""
 
-DEPEND=""
 RDEPEND="${DEPEND}
```

```suggestion
RDEPEND="
```
This is also not needed.